src/ray/raylet/raylet.cc

```diff
@@ -66,8 +66,17 @@ Raylet::Raylet(boost::asio::io_service &main_service, const std::string &socket_
       object_directory_(
           RayConfig::instance().ownership_based_object_directory_enabled()
               ? std::dynamic_pointer_cast<ObjectDirectoryInterface>(
-                    std::make_shared<OwnershipBasedObjectDirectory>(main_service,
-                                                                    gcs_client_))
+                    std::make_shared<OwnershipBasedObjectDirectory>(
+                        main_service, gcs_client_,
+
+                        [this](const ObjectID &obj_id) {
+                          rpc::ObjectReference ref;
```

Comment:
TODO to return error to the client instead (putting the error in the object store is a bit sketchy if there's not enough memory).

Reply:
Done